config/app.php

```diff
@@ -13,7 +13,7 @@
     |
     */
 
-    'name' => env('APP_NAME', 'Laravel'),
+    'name' => env('APP_NAME', 'Laravel API REST'),
```

Comment:
Deberia ser blog api rest
```suggestion
    'name' => env('APP_NAME', 'Laravel BLOG API REST'),
```